winkmem/manifest.json

```diff
@@ -0,0 +1,29 @@
+{
+  "categories": [
+    "os & system"
+  ],
+  "creates_events": false,
+  "display_name": "Windows Kernel Memory",
+  "guid": "ce9fc6d0-5e12-4570-93bb-bf58894996ba",
+  "is_public": false,
```

Comment:
```suggestion
  "is_public": true,
```

Reply:
@alai my understanding is that this shouldn't be set to true until the integration is available in an Agent release, and it won't be available until 7.35 (which will probably come out in march).  Is that correct?

Reply:
Ahh, good to know!